### DISCUSSION TITLE
Password Grant Flow | RFC6749 1.3.3

### DISCUSSION THREAD

**Discussion by SamuelMarks** [zitadel/zitadel#8797] [Q&A]
How do I do the password grant flow in Zitadel?

Specifically I want to send {email,password} and get {access_token,refresh_token,…} back. E.g., in an HTTP POST.

I have implemented RFC6749 in both Python and [separately] TypeScript in the past, and understand that recently 1.3.3 has been deprecated. If you don't plan on implementing this feature, what's the alternative, that doesn't require some OTP sent out-of-band [e.g., to email]?

**Comment by hifabienne** [comment]
Hi @SamuelMarks 
I want to understand you use case a little better.
Who does want to authenticate? Is that a machine account? Or human user?
For human users we only allow UI login flows at the moment.
For machine accounts you have multiple possibilities.
- Personal Access Token (PAT)
- Client Credential Flow
- JWT Profile
https://zitadel.com/docs/guides/integrate/service-users/authenticate-service-users

**Comment by SamuelMarks** [comment]
Auth for human users via Kotlin Multiplatform or Angular.

Human users may want to register/login with email, social, or Google.

Ok so sounds like Zitadel isn't the right project for me. Do you have alternative open-source recommendations?

**Comment by hifabienne** [comment]
Maybe we missunderstood each other.
Human users can register login with email and social logins such as Google.
ZITADEL login is built with standards. We recommend using the OIDC standard, but also SAML is supported.
We do have a step by step guide here on how to integrate ZITADEL login to your Angular application: https://zitadel.com/docs/examples/login/angular

**Comment by hifabienne** [comment]
I will transform the issue into a discussion as it is more of a question at the moment.
